test/generators/csharp/presets/__snapshots__/CommonPreset.spec.ts.snap

```diff
@@ -163,3 +163,69 @@ exports[`CSHARP_COMMON_PRESET should render GetHashCode support function 2`] = `
 
 }"
 `;
+
+exports[`CSHARP_COMMON_PRESET should render short accessors 1`] = `
+"public class Test {
+  private string stringProp;
+  private float? numberProp;
+  private NestedTest objectProp;
+  private Dictionary<string, dynamic> additionalProperties;
+  private Dictionary<string, string> sTestPatternProperties;
+
+  public string StringProp 
+  {
+    get { return stringProp; }
+    set { stringProp = value; }
```

Comment:
Should this not be short accessors? 🤔

Reply:
Should be, but now it's removed anyways :)